### ISSUE TITLE
Add CSS Grid options

### ISSUE THREAD

**Issue by tomusborne** [tomusborne/generateblocks#968]
Our blocks would benefit from a CSS Grid system.

**Comment by eastriverstudio** [comment]
@tomusborne This sounds great, I guess you might have plans to add the extra options to the default container type, but back when you originally announced flex controls for containers I wondered what would become of the existing grid block and thought this might be a good transition for it. As we can now achieve what the grid block did with just containers and flex it makes sense (to me at least!) that the grid block could be transitioned to be powered by CSS grid and then have a slightly different offering and purpose to the root container with its flex options.

**Comment by pagibson** [comment]
@tomusborne , any visions for this in the future? Currently I'm creating a global style for every one-off container that I need to set up a special grid for, because Display > Grid is only available in global styles but not the block options.

**Comment by tomusborne** [comment]
@pagibson Yes, everything in our Global Styles is coming to our new local blocks in 2.0: https://github.com/tomusborne/generateblocks/pull/1268

**Comment by pagibson** [comment]
Whoaaaaa so exciting! Thanks for such a pleasant, trim WP theme and plugin (:

**Comment by diggeddy** [comment]
Complete 2.0
